### PR TITLE
How we support you page

### PR DESCRIPTION
**Pull request by raoufswe**
- adds "How we support you" page

### CODE REVIEW

File: apps/pie-microsite/src/_11ty/shortcodes/list.js [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
 const listTypes = require('../../_data/listTypes');
+const pieIconsSvg = require('../filters/pieIconsSvg');
+const pieDesignTokenColours = require('../filters/pieDesignTokenColours');
 
 /**
  * A List HTML component – takes an array of list items and turns them into a marked-up list
- * @param {string} type - Type of list: ordered, pill
- * @param {string[]} items - An array of list items
+ * @param {string} type - Type of list: ordered, pill, icon
+ * @param {string[] | {iconName: 'string', iconFill?: 'string', content: 'string'}[]} items - An array of list items
```
Comment (jamieomaguire) [inline apps/pie-microsite/src/_11ty/shortcodes/list.js:9]: please could we add some snapshot tests for this shortcode?

Comment (jamieomaguire) [inline apps/pie-microsite/src/_11ty/shortcodes/list.js:9]: could we ask users for a colour token name rather than a fill value?

Comment (jamieomaguire) [reply apps/pie-microsite/src/_11ty/shortcodes/list.js:9]: check line 1 of `header.njk` for how we already do something similar

Comment (jamieomaguire) [reply apps/pie-microsite/src/_11ty/shortcodes/list.js:9]: sorry - I think you've actually done this - ignore if so :D

File: apps/pie-microsite/src/content/pages/designers/how-we-support-you.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+As a result of that, we’re often asked how PIE designers are usually involved when working together with business area designers. To help describe it, we’ve put together a list of the things we do and don’t as part of our role:
+
+### What we do:
+
+{% list {
+    type: listTypes.icon,
+    items: [
+        {
+          iconName: 'CheckCircleFilledSmall',
```
Comment (jamieomaguire) [inline apps/pie-microsite/src/content/pages/designers/how-we-support-you.md:81]: do we anticipate needing a different type of icon per list item? if not we could reduce some duplication here and just specify what icon to use for the entire list

File: apps/pie-microsite/src/content/pages/designers/how-we-support-you.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+          iconFill: 'support-positive',
```
Comment (jamieomaguire) [inline apps/pie-microsite/src/content/pages/designers/how-we-support-you.md:82]: same for fill

**Comment by jamieomaguire** [comment]
Screenshot 2023-02-03 at 13 01 26

@raoufswe this is a question to relay back to Jordi please - the flow chart image I've pasted above is extremely difficult to read. I think we need to get something more legible. I'm trying to read this on a 16 inch macbook pro :)

File: apps/pie-microsite/src/assets/styles/components/_list.scss [hunk 4]
```diff
@@ -51,3 +51,17 @@
...
+        display: flex;
+        gap: f.spacing();
+        margin-bottom: f.$spacing-02;
+
+        & > svg {
+            flex-shrink: 0;
+            margin-top: 2px;
+        }
+    }
+
```
Comment (jamieomaguire) [inline apps/pie-microsite/src/assets/styles/components/_list.scss:66]: NIT: can we remove this line break?

**Review by jamieomaguire — DISMISSED** [review]

**Review by jamieomaguire — APPROVED** [review]

**Comment by jordifornellsJET** [comment]
> Screenshot 2023-02-03 at 13 01 26
> 
> @raoufswe this is a question to relay back to Jordi please - the flow chart image I've pasted above is extremely difficult to read. I think we need to get something more legible. I'm trying to read this on a 16 inch macbook pro :)

This image is for purely for illustrative purposes and just an overview. The paragraph above clearly states "You can check the process closely in our Process file" which is a way for us to redirect them to a place where they can check the process closely and more easily. We want designers to check this Figma file as having another format (like an explanation in paragraphs or lists) will make it less visual and thus harder to understand (there's too many back and forth parts to this process to write it in a lineal way).

**Pull request merged**